DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/Index.cshtml

```diff
@@ -53,7 +53,7 @@
 
     <div class="nhsuk-grid-row">
       <div class="nhsuk-grid-column-full">
-        @if (!Model.Admins.Any()) {
+        @if (!Model.Admins.Any() && Model.FilterBy == null) {

```

Comment:
Fixes a bug for JS version found by Dan where if the user filters down to the point where there are no results then navigates away and back to the page. When the user tries to interact with the filters again, the javascript throws errors because it can't find the elements needed to update by id